pkg/http/core/kubernetes/deploy.go

```diff
@@ -278,3 +207,98 @@ func getListOptions(app string) (metav1.ListOptions, error) {
 
 	return lo, err
 }
+
+func mergeManifests(kubeController kube.Controller, manifests []map[string]interface{}) ([]map[string]interface{}, error) {
+	mergedManifests := []map[string]interface{}{}
+
+	for _, manifest := range manifests {
+		u, err := kubeController.ToUnstructured(manifest)
+		if err != nil {
+			return mergedManifests, err
+		}
+
+		if strings.EqualFold(u.GetKind(), "list") {
+			listElement := kube.ListElement{}
+
+			b, err := json.Marshal(u.Object)
+			if err != nil {
+				return mergedManifests, err
+			}
+
+			err = json.Unmarshal(b, &listElement)
+			if err != nil {
+				return mergedManifests, err
+			}
+
+			mergedManifests = append(mergedManifests, listElement.Items...)
+		} else {
+			mergedManifests = append(mergedManifests, u.Object)
+		}
+	}
+
+	return mergedManifests, nil
+}
+
+func getKubeClientConfig(c *gin.Context, sqlClient sql.Client, account string) (*rest.Config, error, int) {
```

Comment:
For getters in Go I usually see the word `get` not included in the func name (this doesn't apply to HTTP clients that are making a GET request). Rename this to `kubeClientConfig` please.